Kiosk/App/AppSetup.swift

```diff
@@ -30,6 +30,6 @@ class AppSetup {
 
     var needsZipCode: Bool {
         // If we're swiping with the card reaer, we don't need to collect a zip code.
-        return disableCardReader
+        return false
```

Comment:
Well...not exactly. The ZIP code screen is enabled never according to current rules, so this is technically correct. So we can kill this, but probably should kill the original ZIP code screen altogether when we do. I think we can defer that to a lower-priority task.